include/mbedtls/mbedtls_config.h

```diff
@@ -2160,6 +2160,14 @@
  */
 #define MBEDTLS_DEBUG_C
 
+/**
+ * \def MBEDTLS_INST
```

Comment:
This is a cross-module feature, not a module, so it should go in the “mbed TLS feature support” section. And in alphabetical order please.

Reply:
Renamed both.

Reply:
Moved.